src/form.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+
+export default function Form(props) {
+  return (
+    <div className="card mx-auto mt-5 text-center" style={{width: "32rem"}}>
```

Comment:
It's a good idea to define your `styles` outside of your function/render definition so that the objects are not created anew each time the function/render is called.